cmd/api/src/api/v2/apiclient/ingest.go

```diff
@@ -89,7 +104,7 @@ func (s Client) SendCompressedFileUploadData(jsonFile io.Reader, id int64) error
 		return fmt.Errorf("failed to close gzip writer: %w", err)
 	}
 
-	request, err := s.NewRequest(http.MethodPost, fmt.Sprintf("api/v2/file-upload/%d", id), nil, io.NopCloser(&body))
+	request, err := s.NewRequest(http.MethodPost, fmt.Sprintf("api/v2/file-upload/%d", id), nil, io.NopCloser(&body), http.Header{headers.ContentType.String(): []string{mediatypes.ApplicationJson.String()}})
```

Comment:
shouldn't this be the mediatype for gzip?

Reply:
Nope, zip file != gzip file